.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and Test Gem 
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        ruby-version: ['3', '2']
```

Comment:
I don't think support for ruby 2 is valuable, since it hit end of life in [March 2023](https://www.ruby-lang.org/en/news/2023/03/30/ruby-2-7-8-released/)
I'd even prefer if we tested on 3.1, 3.2 and 3.3 separately

Reply:
Thanks for the comment! I agree with removing Ruby 2.x from the all PRs CI. However, since many systems are still using Ruby 2.7 and the gem is quite simple to maintain, I'd like to continue including 2.x in the CI for version releases.

If maintaining support for Ruby 2.x ever becomes too burdensome, we can revisit this decision and drop support at that point 😬